### ISSUE TITLE
Connection to Postgres over SSH fails after upgrade to version 228 or later version of tableplus

### ISSUE THREAD

**Issue by quanta** [TablePlus/TablePlus#1275]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgreSQL 9.6

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 228, 233 and 240

3. The steps to reproduce this issue:

1.  Create connection to PostgreSQL 9.6 database over SSH using either username/login or SSH keyfile on macOS Mojave 10.14.5.  
2.  Try 'test' or 'connect' and note that address and port fields change to red background and connection fails
3.  Downgrade to version 222 of TablePlus and connection succeeds.  

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Thanks, @quanta let me check, it looks like a backward compatible bug.

**Comment by huyphams** [comment]
Hi please try this build and let me know the result: https://www.dropbox.com/s/0i0ofnhi88vlxz8/TablePlus.dmg?dl=0

**Comment by quanta** [comment]
Hi, I tried this one, but I’m still seeing the same problem.

**Comment by digitalpardoe** [comment]
I'm also having this issue, the old debug build published on one of the help pages works fine but the latest build (through SetApp) fails with `Invalid host private key`.

**Comment by huyphams** [comment]
Hi, let me update the debug tool with the new lib.

**Comment by huyphams** [comment]
Hi @digitalpardoe @quanta I updated the debug build with the new lib, please check it: https://www.dropbox.com/s/qdijo59365vsl97/TablePlus-Debug.dmg?dl=0
@quanta here is the blog: https://tableplus.com/blog/2018/10/failed-to-connect-via-ssh-tableplus.html

**Comment by digitalpardoe** [comment]
I've tried with the updated debug build, I can no longer connect, I get the same error as the release build.

**Comment by huyphams** [comment]
Please follow the instructions and send me the log to email nick@tableplus.com it helps me sort out the issue, thank you guys

**Comment by attempted** [comment]
Same issue here, had to downgrade in order to connect.

**Comment by huyphams** [comment]
Hey guys, is it still happening on the latest build 246?

**Comment by quanta** [comment]
Yes, it’s still happening for me.

**Comment by digitalpardoe** [comment]
Yep, still happening for me too.

**Comment by digitalpardoe** [comment]
This is still happening in 252.

**Comment by attempted** [comment]
Any updates on this?

**Comment by huyphams** [comment]
I still can't reproduce the issue. I've updated the debug build to the latest release. Could you guy follow this blog: https://tableplus.com/blog/2018/10/failed-to-connect-via-ssh-tableplus.html and send me the SSH log, thank you!
my email is nick@tableplus.com

**Comment by attempted** [comment]
@huyphams 

Running it with the debug build actually connects fine. The issue for me in the production version is  even when testing the connection the SSH sever and port are highlighted as red for some reason:

<img width="1065" alt="Screen Shot 2019-09-27 at 8 13 28 PM" src="https://user-images.githubusercontent.com/29066530/65808605-6a1fed80-e163-11e9-9aba-3585c4b8ce80.png">

**Comment by huyphams** [comment]
it's weird because they have the same version of SSH. How about this build @attempted 
https://www.dropbox.com/s/0i0ofnhi88vlxz8/TablePlus.dmg?dl=0

**Comment by attempted** [comment]
@huyphams Same issue with that build. Both the server and port light up as red when testing.

**Comment by attempted** [comment]
@huyphams is there anything else I can do to help you bug test?

**Comment by huyphams** [comment]
I've updated the debug build to the lasted release, please try it @attempted 
https://tableplus.com/blog/2018/10/failed-to-connect-via-ssh-tableplus.html
Send me the log via nick@tableplus.com
Thanks!

**Comment by attempted** [comment]
No issues connecting on the debug build again. Do you still want the log? @huyphams

**Comment by huyphams** [comment]
I end up merged to build into the new release @attempted you can enable debug mode in the new beta build here https://www.dropbox.com/s/0i0ofnhi88vlxz8/TablePlus.dmg?dl=0 by going to menu Help > Enable SSH Debug log. Please let me know the result

**Comment by attempted** [comment]
@huyphams so when I try to test the connection with the build you linked, again the server and port light up red (like in the screenshot I shared above). But when I enable the SSH log, in command line it asks for my private key passphrase, which I enter, then everything connects normally.

So since this is both failing and working in the same exact build, could there be an issue with the path to the private key?

**Comment by huyphams** [comment]
I absolutely have no idea what is going on 🦀 here is the code that handles the debug log, just 3 lines of code that made the build work?
<img width="536" alt="Screen Shot 2019-10-10 at 10 25 20 PM" src="https://user-images.githubusercontent.com/3654622/66617790-ef1bf580-ebac-11e9-8b46-af229bb23bb9.png">

**Comment by huyphams** [comment]
Meanwhile, it just enabled some log at the beginning, you can keep it enable as a workaround, hey @quanta @digitalpardoe could you please try the build above with SSH enable, thanks!

**Comment by huyphams** [comment]
Wait, so you entered the password in the command line not from the app? @attempted

**Comment by attempted** [comment]
@huyphams right, when enabling the SSH log my command line prompts me to enter my private key passphrase, not the app.

**Comment by attempted** [comment]
@huyphams was there something else I should have done?

**Comment by huyphams** [comment]
I think I knew the issue, somehow it failed to ask the password, I will come back to you later, could you please email me at nick@tableplus.com ? I think we should not comment here without a solution, it could be annoying for people who subscribe to this issue.
